tests/metrics/network/network-metrics-cpu-consumption.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Description:
+#  Measures cpu % consumption using an inter (docker<->docker) 
+#  network bandwidth using iperf2
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+# Port number where the server will run
+port=5001:5001
+# Image name
+image=gabyct/network
+# Total measurement time (seconds)
+# This is requiered in order to decrease standard deviation
+total_time=16
+# Middle measurement time (seconds)
+# This time is required in at half of the run when
+# server and client are more stable
+middle_time=8
+# Process to track
+process_name="qemu-system-x86_64"
```

Comment:
@devimc the @QEMU_PATH@ can not be used in centos 7.0 https://github.com/01org/cc-oci-runtime/issues/844